packages/@react-spectrum/table/stories/Table.stories.tsx

```diff
@@ -1709,3 +1699,27 @@ export function TableWithBreadcrumbs() {
     </Flex>
   );
 }
+
+export function EmptyStateTable() {
+  let [show, setShow] = useState(false);
+
+  return (
+    <Flex direction="column">
+      <ActionButton width="100px" onPress={() => setShow(show => !show)}>Toggle items</ActionButton>
```

Comment:
If you toggle items in, select all, then toggle items out, the select all checkbox is shown as checked and is keyboard navigable even though it is disabled. Not sure if this is just how the story is written and a non-issue, or maybe it's outside of the scope of this fix.

Reply:
It remains keyboard navigable even if you undo the select all again. As soon as it has gotten focus, that's what TableView will send focus to. This happens with the Column headers as well. Toggle the items, navigate to col 3, then toggle the items off. Tab will continue to send you to Column Header 3

Reply:
https://github.com/adobe/react-spectrum/pull/3422#issuecomment-1235814460

Reply:
Thanks for catching this, I'll probably go the route of clearing the tab index if the grid is empty. One possible issue may be if the collection doesn't properly get an updated tab index but I imagine I could override that as well if need be